src/compiler/scala/tools/nsc/settings/ScalaSettings.scala

```diff
@@ -55,7 +55,7 @@ trait ScalaSettings extends AbsScalaSettings
   // argfiles is only for the help message
   /*val argfiles = */ BooleanSetting    ("@<file>", "A text file containing compiler arguments (options and source files)")
   val classpath     = PathSetting       ("-classpath", "Specify where to find user class files.", defaultClasspath) withAbbreviation "-cp"
-  val d             = OutputSetting     (outputDirs, ".")
+  val d             = OutputSetting     (outputDirs, ".", "Destination for generated classfiles.")
```

Comment:
Instead of overriding the setting, we could pull out the description string into a method here and override that in `doc/Settings`. That would avoid the creation of two `OutputSetting` instances.